opts.go

```diff
@@ -24,7 +24,7 @@ import (
 	"github.com/pkg/errors"
 )
 
-type CNIOpt func(c *libcni) error
+type CNIOpt func(c *libcni) error //nolint: golint // type name will be used as cni.CNIOpt by other packages, and that stutters; consider calling this Opt
```

Comment:
For now, adding a "nolint", but we should consider renaming, keeping the old name as alias, given that this module already is at v1.x.x, so we cannot introduce breaking changes.

I have a branch with that change, but will push separate